lib/internal/error_serdes.js

```diff
@@ -95,6 +107,10 @@ function inspect(...args) {
 let serialize;
 function serializeError(error) {
   if (!serialize) serialize = require('v8').serialize;
+  if (typeof error === 'symbol') {
+    return Buffer.concat([Buffer.from([kInspectedSymbol]),
+                          Buffer.from(inspect(error), 'utf8')]);
```

Comment:
I have a hunch this would perform much better:
```suggestion
    return Buffer.from(StringFromCharCode(kInspectedSymbol) + inspect(error), 'utf8');
```